apps/site/components/withDownloadCategories.tsx

```diff
@@ -13,6 +13,7 @@ const WithDownloadCategories: FC<PropsWithChildren> = async ({ children }) => {
   const t = await getTranslations();
   const releases = await getReleaseData();
 
+  // eslint-disable-next-line react-hooks/rules-of-hooks
   const { pathname } = useClientContext();
```

Comment:
Without this disable:

```plaintext
React Hook "useClientContext" cannot be called in an async function.
eslint[react-hooks/rules-of-hooks](https://reactjs.org/docs/hooks-rules.html)
```

I haven't kept up-to-date with the hullabaloo around async functions and hooks. 🤷